packages/workflow/src/Extensions/ArrayExtensions.ts

```diff
@@ -127,8 +56,8 @@ function unique(value: unknown[], extraArgs: string[]): unknown[] {
 		return value.reduce<unknown[]>((l, v) => {
 			if (typeof v === 'object' && v !== null && extraArgs.every((i) => i in v)) {
 				const alreadySeen = l.find((i) =>
-					// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
-					extraArgs.every((j) => deepCompare((i as any)[j], (v as any)[j])),
+					// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any, @typescript-eslint/no-unsafe-call
+					extraArgs.every((j) => deepEqual((i as any)[j], (v as any)[j])),
```

Comment:
We can get rid of these eslint errors if we change asserted types from any to `Record<string, unknown>`:
```
const alreadySeen = l.find((i) =>
					extraArgs.every((j) =>
						deepEqual((i as Record<string, unknown>)[j], (v as Record<string, unknown>)[j]),
					),
				);
```

The eslint-disable below(`		// eslint-disable-next-line @typescript-eslint/no-unsafe-call`) doesn't seem to be required

Reply:
Nice!